definitions/tools/cnvnator.cwl

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Run CNVnator to calcualte copy number variations in WGS samples"
+
+arguments: ["source", "/opt/root/bin/thisroot.sh", { shellQuote: false, valueFrom: "&&" },  "/bin/bash", "run_cnvnator.sh"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/cnvnator-cwl:0.4"
+    - class: ResourceRequirement
+      ramMin: 20000
+      coresMin: 1
+      tmpdirMin: 10000
+    - class: ShellCommandRequirement
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: "run_cnvnator.sh"
+        entry: |
+          #!/bin/bash
+          set -eou pipefail
+
+          # set vars
+          BAM="$1"
+          BIN_SIZE="$2"
+          CHROMOSOMES="${3//,/ }" # input as csv so it's one var to bash script but CNVnator wants them space separated
```

Comment:
This works, but CWL should be able to handle passing a string containing spaces as a single parameter to the script if desired.

Reply:
yup you're right. Forgot that Cromwell puts quotes around inputs. Switched to 
```
itemSeparator: " "
```
and removed the modification of the input in the bash script. And it worked on my test data.